spec/requests/api/v1/users_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+require 'support/auth_helper'
+include AuthHelper
+
+RSpec.describe "Api::V1::Users" do
+  describe "GET users route" do
+    let!(:user) { FactoryBot.create(:random_user) }
+
+    it 'gets an auth error if accessed without a valid token' do
+      get '/api/v1/users', params: { auth0_id: "provider|1234" }, headers: { "Authorization": "Bearer INVALID_TOKEN" }
+      expect(response.status).to eq(401)
+    end
+
+    it 'gets returns an error if user is not found' do
+      newUser = "provider|5678"
+      get '/api/v1/users', params: { auth0_id: newUser }, headers: auth_header
```

Comment:
How can I stub out the authentication part? 
1. I've thought about converting this request test to a controller test 
2. Is there some other way?

Reply:
I'm not sure I have the most insight to be able to respond with authority, but what I did with the reviews tests:
- the test is `type: :controller`
- and then `controller.stub(:authenticate_user! => true)` actually authenticates the user in the before blocks